### PR TITLE
Updated office address and coordinates in map

### PR DESCRIPTION
**Pull request by jovials146**
## Description
Updated office address

# Implementation
**_office address in contact page_**

Updated office address and map coordinates 

From:
` Infratab Bangalore Pvt Ltd
 #20, 24th Main Road
 2nd phase, JP Nagar 
 Bengaluru-560078, 
Karnataka 560103,India `

To:
` 91 springboard,3rd Floor,Padmavathi Complex,
 80 Feet Rd, Koramangala 8th Block, 
  Koramangala, Bengaluru-560095`

## Screenshots
Artboard – 1

## Impacted Areas
List related PRs from other repositories
- No impact on other repositories

List expected works in other repositories
- No work in other repositories

### Checklist
- [x] Set proper milestone and labels
- [x] Code compiles correctly
- [x] Lint pass locally with my changes
- [x] Extended the README/documentation as necessary
- [x] Test Summary - https://infratab.atlassian.net/wiki/spaces/TC/pages/361758769/Updated+the+office+address

### CODE REVIEW

**Comment by priyanka1110** [comment]
@jovials146 
- Update the title for the PR. Check this [PR](https://github.com/Infratab/Website-IN/pull/27) for reference 
- What changes you have done in the descriptions 
- Add Test summary. Please [refer](https://infratab.atlassian.net/wiki/spaces/TC/pages/277643303/27+Updated+the+bangalore+office+address) this Test summary for the reference.

**Comment by priyanka1110** [comment]
@jovials146 Please make both the before and after screenshot to be same.

**Comment by jovials146** [comment]
@priyanka1110 updated the PR as per comments

File: contact.html [hunk 1]
```diff
@@ -217,9 +211,9 @@ <h5>Contact</h5>
   <script src="js/vendors/jquery-2.1.1.min.js"></script>
   <script src="js/vendors/materialize.min.js"></script>
   <script src="js/vendors/sidebar.js"></script>
-  <script src="dist/js/main.js"></script>
```
Comment (priyanka1110) [inline contact.html:220]: @jovials146 Why did you remove the dist folder?

**Comment by priyanka1110** [comment]
In this PR lot required code has been removed so closing this PR and will create one PR for the address update.
